File: fizzBuzz.rb
def fizzBuzz (n)
  case 0
    when n % (3 * 5)
      'fizzbuzz'
    when n % 5
      'buzz'
    when n % 3
      'fizz'
    else
      n
  end
end

n = 1
until n > 100
  puts fizzBuzz(n)
  n += 1
end

File: fizzBuzz.js
fizzBuzz = function(n) {
    switch(true){
        case n % 15 == 0:
            return "fizzbuzz";
            break;
        case n % 5 == 0:
            return "buzz";
            break;
        case n % 3 == 0:
            return "fizz";
            break;
        default:
            return n;
    }
}

var result = "";
for(var i = 1; i <= 100; i++){
    result += fizzBuzz(i) + " ";
}
console.log(result);